ovs/extensions/generic/heartbeat.py

```diff
@@ -32,12 +33,26 @@
 
 celery_path = OSManager.get_path('celery')
 worker_states = check_output("{0} inspect ping -b {1} --timeout=5 2> /dev/null | grep OK | perl -pe 's/\x1b\[[0-9;]*m//g' || true".format(celery_path, amqp), shell=True)
+domain = check_output(["hostname", "-d"]).splitlines()[0]
 routers = StorageRouterList.get_storagerouters()
 for node in routers:
     if node.heartbeats is None:
         node.heartbeats = {}
-    if 'celery@{0}: OK'.format(node.name) in worker_states:
-        node.heartbeats['celery'] = current_time
+    for s in worker_states.splitlines():
```

Comment:
Never use single character variables please, make sure the variables have a meaningful name